gridData/core.py

```diff
@@ -363,10 +381,11 @@ def _export_python(self, filename):
         with open(filename, 'wb') as f:
             cPickle.dump(data, f, cPickle.HIGHEST_PROTOCOL)
 
-    def _export_dx(self, filename):
+    def _export_dx(self, filename, type=None, **kwargs):
```

Comment:
kwargs can be removed now.

Reply:
Technically yes; practically it says that all `_export_*()` methods need to be able to take all kwargs and only process the ones that are needed. 

That's also the reason for `_export_python()` just using `**kwargs` and not doing anything with it.